src/themes/default/components/core/blocks/Checkout/Tooltip.vue

```diff
@@ -0,0 +1,45 @@
+<template>
```

Comment:
shouldn't the tooltip be a global component?

Reply:
I moved tooltip to `components/core/Tooltip.vue` but I'm not sure if this is what you expect. 
Or should I use [globally registered component](https://vuejs.org/v2/guide/components.html#Global-Registration) instead?